AISKU/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/applicationinsights-web",
-    "version": "2.6.5",
+    "version": "2.7.0-beta.1",
```

Comment:
Just curious - does this not bump to 3.x?

Reply:
Nope, I did a full analysis of the differences between TS 2 and Ts 4 and there are no real breaking changes, so there is no need to bump up to v3

Reply:
I see, you mentioned that there's no breaking changes. I thought that was the reason for update, missed the part of version names. Sounds good.